netsuite/10.0/modules/ROOT/pages/netsuite-call-restlets-example.adoc

```diff
@@ -0,0 +1,159 @@
+= Call NetSuite RESTlets Example
+
+This example shows how to call NetSuite RESTlets that are deployed in NetSuite instances to:
+
+* Create a customer record
+* Retrieve a customer record
+* Delete a record
```

Comment:
a customer record?

Reply:
I sort of expected the subtopics to be create, retrieve, and delete rather than deploy, import, and about the flows.
Do we have a disconnect between the intro to the topic set and the topics within that set?